packages/x-scheduler/src/primitives/use-event-calendar/EventCalendarStore.selectors.ts

```diff
@@ -55,4 +55,23 @@ export const selectors = {
       return state.occurrencePlaceholder;
     },
   ),
+  isCreatingNewEventInDayGridCell: createSelector((state: State, day: SchedulerValidDate) => {
+    const ph = state.occurrencePlaceholder;
+    if (!ph || ph.surfaceType !== 'day-grid' || ph.eventId != null) {
+      return false;
+    }
+    return state.adapter.isSameDay(day, ph.start);
+  }),
+  isCreatingNewEventInTimeRange: createSelector(
+    (state: State, start: SchedulerValidDate, end: SchedulerValidDate) => {
+      const ph = state.occurrencePlaceholder;
```

Comment:
What happens if the placeholder is between two days ? 
It could return true for 2 columns and call `startEditing` twice?